src/components/molecules/AdminSpaceCard/AdminSpaceCard.tsx

```diff
@@ -37,10 +37,18 @@ export const AdminSpaceCard: React.FC<AdminSpaceCardProps> = ({ venue }) => {
         <div className={logoClasses} />
         <AdminShowcaseSubTitle>{venue.name}</AdminShowcaseSubTitle>
       </div>
-      <ButtonNG linkTo={adminNGVenueUrl(venue.id)} variant="primary">
+      <ButtonNG
+        linkTo={adminNGVenueUrl(venue.slug)}
+        disabled={!venue.slug}
```

Comment:
The script didn't populate all of the slugs for some reason, and we need to check why, until then this should prevent the users from opening pages when slug is missing.

Reply:
@denisdimitrov now I can't open my existing spaces as well